.github/workflows/python-tests.yml

```diff
@@ -31,15 +31,8 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
```

Comment:
Did you remove the `pip` upgrade because we're now using Python 3.9 to build the wheels?

Reply:
cibuildwheel does its own upgrade of build tools (it even installs another python on MacOS).

Reply:
Okay, great.